src/tests/ftest/util/daos_core_base.py

```diff
@@ -91,10 +66,9 @@ def run_subtest(self):
                                        '/run/daos_tests/num_replicas/*')
         args = self.params.get("args", '/run/daos_tests/Tests/*', "")
 
-        cmd = "{} -n {} {} {} -s {} -{} {}".format(self.orterun, num_clients,
-                                                   self.orterun_env,
-                                                   self.daos_test,
-                                                   num_replicas, subtest, args)
+        cmd = "{} -n {} {} -s {} -{} {}".format(self.orterun, num_clients,
```

Comment:
This still needs the D_LOG_FILE env set.  You should be able to set it here:
```
"-x D_LOG_FILE={}".format(self.client_log)
```

Reply:
This should stay as per comment above.

Reply:
Summary of our offline discussion/tests using daos_core_test.py:
- test 1: using self.orterun_env with both ortrerun daos_server & orterun daos_test
-> renamed server & client logs exist (expected)
- test 2: using self.orterun_env with only ortrerun daos_test
-> renamed server & client logs exist (expected)
- test 3: not using self.orterun_env with either ortrerun command
-> only renamed server logs exist; no client logs (expected)
- test 4: using self.orterun_env with only ortrerun daos_server
-> renamed server & client logs exist (unexpected)
- test 5: using self.orterun_env with ortrerun daos_server + unique log with orterun daos_test
-> renamed server & unique daos_test logs exist

Tests 3, 2, 5, and possibly 1 would seem to indicate that the "-x D_LOG_FILE=self.client_log" is required.  Test 4, however, supports what you are seeing.  Is it possible that daos_test is somehow using the D_LOG_FILE env from daos_server?  When I ran my experiments I did have a D_LOG_FILE env set in my ~/.bashrc, but I never saw logs set using that assigned name.